src/main/groovy/com/brightsparklabs/gradle/docs/DocsPlugin.groovy

```diff
@@ -452,9 +452,15 @@ public class DocsPlugin implements Plugin<Project> {
 
                 asciidoctorj {
                     // 'book' adds a cover page to the PDF
-                    options doctype: 'book'
+                    Map<String,Object> pluginOptions = [ "doctype" : 'book' ]
```

Comment:
Define a constant in the file called `DEFAULT_ASCIIDOCTOR_OPTIONS = [ "doctype" : 'book' ]`
Then refer to that here and in the javadoc comments later on. Either copy it or put all:
```
 Map<String,Object> pluginOptions = [:].putAll(DEFAULT_ASCIIDOCTOR_OPTIONS)`
```

NOTE: ASCIIDOCTOR not ASCIIDOC as the options pertain to the ASCIIDOCTOR tool